docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,99 @@
 
-Tell us about your stack. What languages and libraries are you using, and which versions of these?
+## The 'where'
 
-**Additional information**
+Tell us about the environment the problem happened in: 
+-	In a cluster, let us know the namespace and cluster name
+- With a stack, let us know what languages and libraries you used, including which versions
+- With a tool, please specify which platform tool
+- With a web application, please specify which browser, browser version, and device
+     - A Mac, a Windows machine, an iPhone, an Android, or another device?
 
-Also explain what device you were using when you experienced the problem. Were you using a Windows machine, a Mac, an iPhone, an Android or something else? If you were using a web application, what browser were you using? Provide as much detail as possible, including version numbers.
+ ## The 'when'
 
----
-## What are you trying to do?
-
- Explain what task you were trying to accomplish.
- 
--  How you were doing it, and why it was important. The more detail you can provide the better. 
-- Let the community know what you were trying to do and what outcome you had expected.
-
----
-## What is the problem?
+ Timestamps help! Please consider letting us know: 
+- The date
+- The time, including time zone if different from P.S.T. 
+- If working on a legacy task, the timestamp of the original work
 
-Now that the community knows what was supposed to happen, tell them what went wrong. Include descriptions of
 
-- How the problem occurred
-- What the problem looks like and how it’s behaving. For example, if a screen is failing to load properly, explain how it looks and why it’s not correct. 
-- Provide as much detail as possible. If you received an error message, include a screenshot of the message with your question. 
+## Post the question
 
-##  What have you done to fix the problem?
+When you finish writing the detailed question, please revisit [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) to make sure you're being efficient. Then, post it in the best Rocket.Chat channel. 
 
-The results of your initial troubleshooting efforts can be very useful to the community when they’re trying to determine the cause of your issue. Make sure you include any steps you’ve already taken to resolve the issue, and include as much detail as possible. 
+If you're not being proactive about your steps, then the community might miss the question in favour of responding to the ones specific for the channel. 
 
-If you’ve been following instructions on a StackOverflow thread, provide a link to the thread. If you’ve been reading documentation about the problem, provide a link to the documentation. 
+If unsure of where to post or other questions and/or concerns, connect with us at developer.experience@gov.bc.ca.
```

Comment:
first step should be to check the "Rocket.Chat channel descriptions" page.

Reply:
> first step should be to check the "Rocket.Chat channel descriptions" page.

Updated with: "If unsure of where to post or other questions and/or concerns, check out the [Rocket.Chat channel descriptions](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/rocketchat/rocketchat-channel-descriptions/). If nothing there helps, then  connect with us at developer.experience@gov.bc.ca.'